src/main/kotlin/org/jetbrains/research/ml/dataset/anonymizer/transformations/anonymization/JvmAnonymizationVisitor.kt

```diff
@@ -46,7 +46,15 @@ open class JvmAnonymizationVisitor(file: PsiFile, val anonymizer: JvmElementAnon
             try {
                 processor.renameElement(definition, newName, usages, null)
             } catch (e: IncorrectOperationException) {
-                log.info("Cannot perform the renaming for element $definition with new name $newName")
+                log.info(
+                    "IncorrectOperationException: Cannot perform the renaming for " +
+                        "element $definition with new name $newName"
+                )
+            } catch (e: NullPointerException) {
+                log.info(
+                    "NullPointerException: Cannot perform the renaming " +
+                        "for element $definition with new name $newName"
+                )
```

Comment:
Looks a little bit strange, maybe try catching Exception in general (like `catch(e: Exception)`) and then use `when` block to specify that for `NullPointerException` and `IncorrectOperationException` you want to print the message with `e.toString` as its name? or we can use `.javaClass.simpleName`

```
when(e) {
            is IllegalStateException, is NullPointerException -> log.info(...)
            else -> throw e
        }
```